include/models/overwrite.hpp

```diff
@@ -0,0 +1,33 @@
+#ifndef _overwrite_hpp_
+#define _overwrite_hpp_
+
+#include <models/entity.hpp>
+
+namespace disccord
+{
+    namespace models
+    {
+        class overwrite : public model

```

Comment:
ah okay, so everything w/ a snowflake id related to that specific object should inherit entity?